src/main/java/org/researchspace/sail/rest/AbstractRESTWrappingSailConnection.java

```diff
@@ -62,9 +66,8 @@ public AbstractRESTWrappingSailConnection(AbstractServiceWrappingSail sailBase)
      */
     protected Response submit(RESTParametersHolder parametersHolder) {
         try {
-            Client client = ClientBuilder.newClient();
-            WebTarget targetResource = client.target(getSail().getUrl()).property(ClientProperties.FOLLOW_REDIRECTS,
-                    Boolean.TRUE);
+            WebTarget targetResource = this.getSail().getClient().target(getSail().getConfig().getUrl())
+                    .property(ClientProperties.FOLLOW_REDIRECTS, Boolean.TRUE);
```

Comment:
just an observation fyi, it is not going to follow redirect from http->https, and will just fail. Which, I believe is OK. See  https://stackoverflow.com/questions/21778393/jersey-jax-rs-client-2-5-follow-redirect-from-http-to-https and https://stackoverflow.com/questions/1884230/httpurlconnection-doesnt-follow-redirect-from-http-to-https